karma.conf.js

```diff
@@ -173,5 +173,90 @@ module.exports = function (config) {
     options.detectBrowsers.enabled = false;
     options.browsers = process.env.BROWSERS.split(" ");
   }
+
+  if (process.env.GITHUB_WORKFLOW) {
+    const localPlugins = [
+      { "reporter:respec-github-action": ["type", GithubActionReporter] },
+    ];
+    options.reporters.push("respec-github-action");
+    options.plugins = ["karma-*"].concat(localPlugins);
+  }
+
   config.set(options);
 };
+
+function GithubActionReporter() {
```

Comment:
I really want to move this to a separate file, something like `tools/github-action-reporter.js`. Is there a reason to keep it here?

Reply:
Not a good reason. Just wanted to keep it within karma as it's related. Will move.